pkg/kds/server/status_sink.go

```diff
@@ -64,6 +76,7 @@ func (s *zoneInsightSink) Start(stop <-chan struct{}) {
 		select {
 		case <-ticker.C:
 			flush()
+			time.Sleep(s.flushBackoff)
```

Comment:
I guess this `time.Sleep` is done in order to guarantee delay between 2 `flush()` and avoid attacking Kubernetes API too often. Maybe it does make sense to use `rate.NewLimiter` like in the insights resyncer to have the same approach across the project. What do you think?

Reply:
discussed offline that backoff have more sense in this case. To achieve similar behavior we would need to get rid of the ticker